Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1484,6 +1484,14 @@ private async Task<AuthenticationResult> AuthenticateNewSessionInternal(Authenti
                 throw new SecurityException("User is not allowed access from this device.");
             }
 
+            int sessionsCount = Sessions.Count(i => i.UserId.Equals(user.Id));
+            int maxActiveSessions = user.MaxActiveSessions;
+            _logger.LogInformation("Current/Max sessions for user {User}: {Sessions}/{Max}", user.Username, sessionsCount, maxActiveSessions);
```

Comment:
Probably only log this if the max sessions is set

Reply:
I debated this, but I think the information is important and useful regardless. I've long thought we log far too much useful data like this.